server/trillian_log_server/main.go

```diff
@@ -68,6 +70,9 @@ func main() {
 	glog.CopyStandardLogTo("WARNING")
 	glog.Info("**** Log RPC Server Starting ****")
 
+	// Enable dumping of metrics to the log at regular interval.
+	go metric.DumpToLog(context.Background(), 10*time.Second)
```

Comment:
I'd like to have expvars for them, yes, I'll build on this.

I'll add a flag now, though, good idea.